applications/teletype/src/teletype_templates.erl

```diff
@@ -196,7 +202,7 @@ templates_source(TemplateId, AccountId, AccountId) ->
     case fetch_notification(TemplateId, AccountId) of
         {'ok', Template} ->
             templates_source_has_attachments(TemplateId, AccountId, AccountId, Template);
-        {'error', 'not_found'} -> ?KZ_CONFIG_DB;
```

Comment:
This is not necessary, because `fetch_notification` also fetch `Reseller` and `system_config` notification templates.
If `fetch_notification` returned `not_found`, than notification template is not exist in `system_config`  too. And returning `system_config` will be false response.